src/Http/Controllers/Api/Geojson/VolumeImagesController.php

```diff
@@ -12,10 +13,18 @@ class VolumeImagesController extends Controller
   public function index($id){
     $volume = Volume::findOrFail($id);
     $this->authorize('access', $volume);
-    $images = $volume->images;
 
-    $features = collect($images)->map(function($image){
-      $feature = new \GeoJson\Feature\Feature(new \GeoJson\Geometry\Point([$image->lng, $image->lat]), Array('test'=>"test"));
+    $images = Image::where('volume_id', $id)->with(['annotations'=>function($a){
```

Comment:
I would prefer using `join` to the `with` calls. `with` calls execute one SQL query per call. A query with `join` needs to be executed only once. You can debug SQL queries (and see how many queries your code produces) with:

```php
DB::listen(function ($query) {
   dump($query->sql);
});
```

Example with join:

```php
$images = $volume->images()->select('id', 'lat', 'lng');

$labels = $volume->images()
   ->join('annotations', 'annotations.image_id', '=', 'images.id')
   ->join('annotation_labels', 'annotation_labels.annotation_id', '=', 'annotations.id')
   ->join('labels', 'labels.id', '=', 'annotation_labels.label_id')
   ->select('images.id', 'labels.name');
```